src/chapter1/section1/Exercise33_Matrix_Operations.java

```diff
@@ -165,15 +164,15 @@ private static double dot(double[] x, double[] y) {
 	//Vector-matrix product
 	private static double[] mult(double[] y, double[][] a) {
 		
-		//Number of rows in matrix must be the same as the number of rows in vector
+		//Number of rows in matrix must be the same as the number of columns in vector
```

Comment:
`columns` is more correct than `rows`, but I would suggest using `elements` as you mentioned on the description of this PR.
```suggestion
		//Number of rows in matrix must be the same as the number of elements in vector
```

Reply:
I agree with your suggestion.